interface/main/messages/messages.php

```diff
@@ -23,614 +24,686 @@
  * @author OpenEMR Support LLC
  * @author Roberto Vasquez <robertogagliotta@gmail.com>
  * @author Rod Roark <rod@sunsetsystems.com>
- * @author Brady Miller <brady.g.miller@gmail.com>
+ * @author Brady Miller <brady@sparmy.com>
+ * @author Ray Magauran <magauran@medfetch.com>
  * @link http://www.open-emr.org
  */
 
-
-
 require_once("../../globals.php");
 require_once("$srcdir/pnotes.inc");
 require_once("$srcdir/patient.inc");
 require_once("$srcdir/acl.inc");
 require_once("$srcdir/log.inc");
 require_once("$srcdir/options.inc.php");
+require_once("$srcdir/classes/Document.class.php");
 require_once("$srcdir/gprelations.inc.php");
```

Comment:
@robertdown ,
I'm trying to get the namespacing here in line with OpenEMR's . The MedExApi namespace has a bunch of really generic naming classes. Does that mean we can't do the above use command?